src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer/Personalizer.csproj

```diff
@@ -0,0 +1,30 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <!-- Please do not move/edit code above this line -->
+
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.CognitiveServices.Personalizer</PackageId>
+    <Description>This client library provides access to the Microsoft Cognitive Services Personalizer APIs.</Description>
+    <Version>1.0.0</Version>
```

Comment:
maybe a preview version as the service is public preview?

Reply:
@dwaijam Please mark the version as `0.8.0-preview`

Reply:
done